docs/styles.md

```diff
@@ -713,6 +715,76 @@ Div objects only
 
 ---
 
+## Extra Support Box
+
+Add a styled container and content for support for English Language Learners and students with disabilities.
+
+**Examples**
+
+Support for English Language Learners
+<div style="text-align: center;">
+    <img src="./static/extrasupportboxell.png" width="750">
+</div>
+
+Support for Students with Disabilities
+<div style="text-align: center;">
+    <img src="./static/extrasupportboxdisabilities.png" width="750">
+</div>
+
+**Availability**
+
+Extra support boxes should only be used in `<div>` elements.
+
+**Usage**
+
+Support for English Language Learners
+```html
+<div class="os-raise-extrasupport">
+  <div class="os-raise-extrasupport-header">
+    <p class="os-raise-extrasupport-title">Support for English Language Learners</p>
+    <p class="os-raise-extrasupport-name">MLR 6 Three Reads: Reading, Listening, Conversing</p>
+  </div>
+  <div class="os-raise-extrasupport-body">
+      <p>
+        Ask students to keep their books or devices closed and display only the image with the opening task statement, without revealing the questions that follow.
+        Use the first read to orient students to the situation by asking students to describe what the situation is about without using numbers (a teacher buys calculators and measuring tapes for her class).
+        Use the second read to identify quantities and relationships; ask students: “What can be counted or measured?”
+        Listen for, and amplify, the important quantities that vary in relation to each other in this situation: number of calculators purchased (for each order), number of measuring tapes purchased (for each order), and total cost of each order.
+        After the third read, reveal the questions that follow and invite students to brainstorm possible strategies to find the solution to the system.
+        This routine helps students interpret the language within a given situation needed to create an equation.
+      </p>
+      <p class="os-raise-text-italicize">Design Principle(s): Support sense-making</p>
+      <p class="os-raise-text-bold os-raise-extrasupport-title">Learn more about this routine</p>
```

Comment:
Per [your comment](https://github.com/openstax/k12-apps-raise/pull/133#issuecomment-1830390570), the re-use of `.os-raise-extrasupport-title` here makes sense to me since UX wants the two to look the same. 

Howver, is the `.os-raise-text-bold` necessary if we're using `.os-raise-extrasupport-title` since the latter is setting `font-weight`? I think the latter will always win for rule specificity anyways :man_shrugging: .

Reply:
Good catch! I'll remove the duplicate bold class.